appengine/standard_python37/bigquery/main_test.py

```diff
@@ -47,3 +47,9 @@ def fake_query(query):
     r = flask_client.get('/')
     assert r.status_code == 200
     assert 'Query Timeout' in r.data.decode('utf-8')
+
+
+def test_results(flask_client):
+    r = flask_client.get('/')
```

Comment:
Does this test pass? I think by default the test client doesn't follow redirects. You'll need to pass in `follow_redirects=True`. See: http://flask.pocoo.org/docs/1.0/testing/#logging-in-and-out

Reply:
Also, this test is now identical to `test_main`. You should update `test_main` to check that the redirect goes where expected.

Reply:
`test_main_timeout` will also need `follow_redirects=True`. You should rename that test to `test_results_timeout`.

Reply:
Renamed the functions as directed and made changes to tests in order for them to pass. Thx!